src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,128 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private static User stepan;
+    private RegistrationServiceImpl registerService = new RegistrationServiceImpl();
+
+    @BeforeAll
+    static void beforeAll() {
+        stepan = new User();
+        stepan.setAge(18);
+        stepan.setId(12L);
+        stepan.setLogin("Stepan");
+        stepan.setPassword("test123");
+    }
+
+    @Test
+    void checkRegisterNull() {

```

Comment:
![Screenshot 2022-06-23 at 08 43 08](https://user-images.githubusercontent.com/60258373/175223815-33406086-499d-484d-a5c7-d5f78846306f.png)
the same for other methods